### PR TITLE
docs: add rockcraft chisel notes

### PR DESCRIPTION
**Pull request by syu-w**

### CODE REVIEW

File: docs/base/part_properties.rst [hunk 1]
```diff
@@ -311,6 +311,15 @@ The packages to install in the staging area for deployment with the build
 products. These provide libraries and executables to support the deployed
 part.
 
+For Rockcraft:
```
Comment (cmatsuoka) [inline docs/base/part_properties.rst:314]: It works regardless of the application in use, any special need to mention Rockcraft? If this is valid only for certain applications we can say so and refer to specific application documentation.

Comment (syu-w) [reply docs/base/part_properties.rst]: OK, I removed Rockcraft, it was said only for rockcraft in its docs.

File: docs/base/part_properties.rst [hunk 2]
```diff
@@ -311,6 +311,14 @@ The packages to install in the staging area for deployment with the build
 products. These provide libraries and executables to support the deployed
 part.
 
+This keyword also support chiselled packages.
```
Comment (tigarmo) [inline docs/base/part_properties.rst:314]:
```suggestion
This keyword also supports `Chisel <https://github.com/canonical/chisel>`_ slices.
```

**Review by tigarmo — APPROVED** [review]
Please don't force-push commits during the reviewwwww
:cry: :sob: :crying_cat_face:

**Review by cmatsuoka — APPROVED** [review]
Thanks!

**Pull request merged**